src/components/institution/AffiliationHierarchy.tsx

```diff
@@ -39,6 +45,10 @@ export const AffiliationHierarchy: FC<AffiliationHierarchyProps> = ({
         )
       )}
     </div>
+  ) : (
+    <ErrorTypography>
+      [{t('error.get_affiliation_name')} {unitUri}]
+    </ErrorTypography>
```

Comment:
jeg jobber med landing page nå, og denne er vel like relevant når vi viser som `commaSeparated`? (Linje 36)

Reply:
commaSeparated er den visningen som brukes på landing page? isåfall - ja! skal fikse 👍